src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1042,6 +1056,19 @@ void RaftPart::statusPolling() {
     }
 }
 
+bool RaftPart::needToCleanupSnapshot() {
+    std::lock_guard<std::mutex> g(raftLock_);
+    return status_ == Status::WAITING_SNAPSHOT &&
+           role_ != Role::LEADER &&
+           lastSnapshotRecvDur_.elapsedInSec() >= FLAGS_raft_snapshot_timeout;
+}
+
+void RaftPart::cleanupSnapshot() {
+    LOG(INFO) << idStr_ << "Clean up the snapshot";
+    std::lock_guard<std::mutex> g(raftLock_);
+    reset();
+    status_ = Status::RUNNING;
```

Comment:
STOPPED ?

Reply:
No,   the logic here is to clean up snapshot state,  and reset status to be running from wait_snapshot